src/SeleniumLibrary/keywords/element.py

```diff
@@ -112,21 +113,42 @@ def page_should_contain_element(self, locator, message=None, loglevel='INFO'):
         The ``message`` argument can be used to override the default error
         message.
 
+        The ``limit`` argument can used to define how many elements the
+        page should contain. When ``limit`` is `None` page can contain one or
+        more elements. When limit is a number, page must contain same number
+        of elements.
+
         See `Page Should Contain` for explanation about the ``loglevel``
         argument.
+
+        The ``limit`` argument is new in SeleniumLibrary 3.0
+
+        Examples assumes that locator matches to two elements.
+        | Page Should Contain Element | name: div_name | limit=1    | # Keyword fails.                  |
+        | Page Should Contain Element | name: div_name | limit=2    | # Keyword passes.                 |
+        | Page Should Contain Element | name: div_name | limit=none | # None is considered one or more. |
+        | Page Should Contain Element | name: div_name |            | # Same as above.                  |
```

Comment:
- Why is `name:` prefix needed? The keyword would search by name (and id) also without it.
- I've used single backticks around keyword names in examples to get linking to work. That's mainly useful when examples refer to other keywords, but using them consistently might be a good idea. What do you think?